.readthedocs.yaml

```
@@ -0,0 +1,9 @@
+version: 2
+build:
+    os: ubuntu-20.04
```

Use the suggested default from https://docs.readthedocs.io/en/stable/config-file/v2.html
```suggestion
    os: ubuntu-22.04
```